### DISCUSSION TITLE
novata

### DISCUSSION THREAD

**Discussion by xxxtriple** [community/community#43274] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

Bom, estou tentando aprender mas é complicado, como faz um pull request para contribuir com outros projetos?

**Comment by CodesdaLu** [comment, marked as answer]
Olá! 
Entao tem um passo a passo muito explicativo espero que te ajude
https://www.webdevdrops.com/como-criar-um-pull-request-no-github-passo-a-passo/

**Reply by xxxtriple** [reply]
aii muito obrigada
